### PR TITLE
Hide some sections in admin product listing to make more mobile friendly

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

**Comment by drbyte** [comment]
Since these pages are styled with Bootstrap, shouldn't we just use the built-in `.hidden-md` class?

Ref: https://getbootstrap.com/docs/3.3/css/#helper-classes

**Comment by scottcwilson** [comment]
LOL I looked and looked for that and couldn't find it.  Will do.

**Comment by drbyte** [comment]
Thanks!

**Comment by Zen4All** [comment]
I am not a fan of putting the hidden-md class here, as it will hide stuf on a 1024px wide screen, which is not necessary. Otherwise great fix.

File: admin/category_product_listing.php [hunk 1]
```diff
@@ -648,12 +648,12 @@ function init() {
                 }
                 ?>
                 <tr onclick="document.location.href = '<?php echo zen_href_link(FILENAME_CATEGORY_PRODUCT_LISTING, zen_get_path($category['categories_id'])); ?>'" role="button">
-                  <td class="text-right"><?php echo $category['categories_id']; ?></td>
-                  <td><?php echo zen_image(DIR_WS_ICONS . 'folder.gif', ICON_FOLDER); ?>&nbsp;<strong><?php echo $category['categories_name']; ?></strong></td>
-                  <td class="text-center">&nbsp;</td>
-                  <td class="text-right"><?php echo zen_get_products_sale_discount('', $category['categories_id'], true); ?></td>
-                  <td class="text-center">&nbsp;</td>
-                  <td class="text-right">
+                  <td class="text-right hidden-md hidden-sm hidden-xs"><?php echo $category['categories_id']; ?></td>
```
Comment (Zen4All) [inline admin/category_product_listing.php]: Id column is not hidden in product, and header.

File: admin/category_product_listing.php [hunk 2]
```diff
@@ -648,12 +648,12 @@ function init() {
...
+                  <td hidden-md hidden-sm hidden-xs><?php echo zen_image(DIR_WS_ICONS . 'folder.gif', ICON_FOLDER); ?>&nbsp;<strong><?php echo $category['categories_name']; ?></strong></td>
```
Comment (Zen4All) [inline admin/category_product_listing.php]: This should be removed

**Pull request merged**